python-checks/src/test/resources/checks/regex/duplicatesInCharacterClassCheck.py

```diff
@@ -81,8 +81,8 @@ def compliant(input):
     re.match(r"[aa", input)  # Check should not run on syntactically invalid regexen
     re.match(r"(?U)[\wä]", input)  # False negative because we don't support Unicode characters in \w and \W
     re.match(r"[[a-z&&b-e]c]", input)  # FN because we don't support intersections
-    re.match(r"(?i)[A-_d-{]", input)  # FN because we ignore case insensitivity unless both ends of the ranges are letters
-    re.match(r"(?i)[A-z_]", input)  # FN because A-z gets misinterpreted as A-Za-z due to the way we handle case insensitivity
+    re.match(r"(?i)[A-_d-{]", input)  # Noncompliant
+    re.match(r"(?i)[A-z_]", input)  # Noncompliant
```

Comment:
Nice, that there were fixed by the latest analyzer-commons version!?

Reply:
Indeed! :)